### PR TITLE
infra: add pr template, codeowners

### PR DESCRIPTION
**Pull request by JacobCoffee**
#### What

Add this PR template
Add CODEOWNERS

#### Closes

-

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Notify @EWDurbin for all opened Issues and Pull Requests
```
Comment (hugovk) [inline .github/CODEOWNERS:1]: Name doesn't match list, no need to repeat names in comment, and it'll only assign for PRs (that touch code), not issues:
```suggestion
# Notify for all opened pull requests
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!--
+By submitting this pull request, you agree to:
+- follow the [PSF's Code of Conduct](https://www.python.org/psf/conduct/)
```
Comment (hugovk) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: Direct link:
```suggestion
- follow the [PSF's Code of Conduct](https://policies.python.org/python.org/code-of-conduct/)
```

**Review by hugovk — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (JacobCoffee) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: These comments were collapsed D: I missed them sorry 😞

Comment (hugovk) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: No problem :)

**Pull request merged**